src/components/auth/Login.jsx

```diff
@@ -44,7 +47,9 @@ const Login = () => {
 
   useEffect(() => {
```

Comment:
Instead of relying `useEffect` to redirect logged in users have you tried conditionally redirecting logged in users? E.g. check `isLoggedIn` on mount and then redirect if `truthy`. This Limits our reliance on `useEffects` which is a good idea.